server/views/pages/overview/partials/timelineTab/timelineTabContents.njk

```diff
@@ -7,30 +7,41 @@
         <div class="moj-timeline">
 
         {% for event in timeline.events %}
-          <div class="moj-timeline__item" data-qa-event-type="{{ event.eventType }}">
-            <div class="moj-timeline__header">
-              <h2 class="moj-timeline__title">{{ event.eventType | formatTimelineEvent }}</h2>
-              <p class="moj-timeline__byline govuk-!-display-none-print">by {{ event.actionedByDisplayName }}, {{ event.prison.prisonName if event.prison.prisonName else event.prison.prisonId }}</p>
-            </div>
-
-            <p class="moj-timeline__date">
-              <time datetime="{{ event.timestamp }}">{{ event.timestamp | formatDate('D MMMM YYYY') }}</time>
-            </p>
-
-            {% if event.eventType == 'ACTION_PLAN_CREATED' or event.eventType == 'GOAL_UPDATED' or event.eventType == 'GOAL_CREATED' or event.eventType == 'MULTIPLE_GOALS_CREATED' %}
-            <div class="moj-timeline__description govuk-!-display-none-print">
-              <p class="govuk-body govuk-!-margin-bottom-0">
-                <a href="/plan/{{ prisonerSummary.prisonNumber }}/view/overview">
-                  {% if event.eventType == 'ACTION_PLAN_CREATED' %}
-                    View {{ prisonerSummary.firstName }} {{ prisonerSummary.lastName | title }}'s learning and work progress
-                  {% elif event.eventType == 'GOAL_UPDATED' or event.eventType == 'GOAL_CREATED' or event.eventType == 'MULTIPLE_GOALS_CREATED' %}
-                    View <span class="govuk-visually-hidden">{{ prisonerSummary.firstName }} {{ prisonerSummary.lastName | title }}'s </span>goals
-                  {% endif %}
-                </a>
+          {% if event.eventType == 'PRISON_ADMISSION' or event.eventType == 'PRISON_TRANSFER' or event.eventType == 'PRISON_RELEASE' %}
```

Comment:
Due to the way prison movement events need to appear (e.g. without the `actionedByDisplayName`), we need a separate section to format them.